package.json

```diff
@@ -34,7 +34,7 @@
     "@formatjs/intl-locale": "^3.3.2",
     "@formatjs/intl-pluralrules": "^5.2.4",
     "@formatjs/intl-relativetimeformat": "^11.2.4",
-    "@gorhom/bottom-sheet": "^4.5.1",
+    "@gorhom/bottom-sheet": "^4.6.4",
```

Comment:
this wasn't strictly necessary (was an attempt to help my struggles with bottom sheet stuff). happy to omit or move to a separate PR